_posts/blog/2018-11-20/2018-11-20-big-discounts-when-You-shop-openbazaar-this-week.markdown

```diff
@@ -0,0 +1,114 @@
+---
+title: "<span class='post-title-extra'>Big Discounts</span> When You Shop OpenBazaar This Week"
+layout: post
+date: '2018-11-20 06:30:00 -0300'
+social_title: 'Big Discounts When You Shop OpenBazaar This Week'
+social_description: 'To celebrate the launch of OpenBazaar 2 in November last year we are sharing goodies and offers all month long! Clustered loosely around Black Friday this week are a bunch of exciting offers some vendors have set up just for you.'
+social_image: Pascal_Boyart_on_OpenBazaar.png
+---
+
+To celebrate [the launch of OpenBazaar 2](https://openbazaar.org/blog/openbazaar-20-is-live/) in November of last year we are sharing goodies and offers all month long with our users! Clustered loosely around Black Friday this week are a bunch of exciting offers some OpenBazaar vendors have set up just for you. 
+
+>Unless otherwise specified, use coupon code **OB2Turns1** to claim the discounts on the stores or items described!
+
+#### Here are some great stores and listings for you to shop this week:
+<br>
+#### DrApis
+_75% off_
+
+![DrApis on OpenBazaar](DrApis_on_OpenBazaar.png "DrApis on OpenBazaar")[DrApis on OpenBazaar](https://openbazaar.com/store/QmYTXDyMNjdUSvqNc88T2VeVF3KdG7PMefnGQKrp9NZ5Tp)
+<br>  
+<br>
+DrApis produces and sells premium raw apiculture products from Portugal. Their artisanal raw honey is produced and extracted using exclusively traditional methods, preserving all the natural vitamins, enzymes, phytonutrients and other nutritional elements produced by bees. Now you can enjoy their high-quality products at their lowest prices _ever._ Don't miss this!
```

Comment:
Is is apiculture or apicultural?

Reply:
They say apiculture in their description!